tests/test_hierarchy_helpers.hpp

```diff
@@ -155,11 +155,19 @@ class DiscontinuousMaterialProperty final : public Coefficient<dim>
         unsigned int const = 0) const override
   {
     auto const one = dealii::make_vectorized_array<double>(1.);
-    unsigned int dim_scale = 0;
-    for (unsigned int d = 0; d < dim; ++d)
-      dim_scale += static_cast<unsigned int>(std::floor(p[d][0] * 100)) % 2;
-
-    return (dim_scale == dim ? 100. * one : 10. * one);
+    auto dim_scale = dealii::make_vectorized_array<double>(0);
+    for (unsigned int i = 0;
+         i < dealii::VectorizedArray<double>::n_array_elements; ++i)
+      for (unsigned int d = 0; d < dim; ++d)
+        dim_scale[i] +=
+            static_cast<unsigned int>(std::floor(100. * p[d][i])) % 2;
+
+    dealii::VectorizedArray<double> return_value;
+    for (unsigned int i = 0;
+         i < dealii::VectorizedArray<double>::n_array_elements; ++i)
+      return_value[i] = (dim_scale[i] == dim ? 100. : 10.);
```

Comment:
Can you assign a `double` to a `VectorizedArray`?

Reply:
Yes, https://www.dealii.org/current/doxygen/deal.II/classVectorizedArray.html#a2a6e8e58f020c57afc202b17f93c0774, but that is not what I am doing.